django_cloud_deploy/cloudlib/database.py

```diff
@@ -241,7 +241,6 @@ def with_cloud_sql_proxy(self,
         Raises:
             DatabaseError: If cloud sql proxy failed to start after 5 seconds.
         """
-        db.close_old_connections()
```

Comment:
This line is deleted, because this line requires the Django environment to be set up. But when running management commands, we do not setup Django environment.